<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>newRPL for Prime G1 Installer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../bitmap/icon.png</normaloff>../../../bitmap/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Step 1 - Select Original firmware folder...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select folder where Connectivity Kit extracted original firmware for Prime G1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firmwareFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="folderBrowseButton">
        <property name="text">
         <string>Browse for folder...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Step 2 - Select newRPL firmware file...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select newRPL firmware file for Prime G1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newrplFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="firmwareBrowseButton">
        <property name="text">
         <string>Browse for file...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Step 3 - Installation...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="proceedButton">
        <property name="text">
         <string>Proceed with installation...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Installation log:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
